### PR TITLE
Add ≡ (equivalent to ==) and ≢ (equivalent to !=)

### PR DESCRIPTION
**Pull request by frithnanth**
## The problem

Two Unicode operators first available in Rakudo v2020.07 don't appear in the table under "Other acceptable single codepoints", as said in #3527 

## Solution provided

This PR updates the table.

### CODE REVIEW

**Comment by coke** [comment]
Missing a negation on the second item in the table?

**Comment by frithnanth** [comment]
Fixed

File: doc/Language/unicode_ascii.pod6 [hunk 1]
```diff
@@ -112,6 +112,8 @@ X<|«>X<|»>X<|×>X<|÷>X<|≤>X<|≥>X<|≠>X<|−>X<|∘>X<|≅>X<|π>X<|τ>X<
   ⊖      |  U+2296   | (^)        |
   ⊍      |  U+228D   | (.)        |
   ⊎      |  U+228E   | (+)        |
+  ≡      |  U+2261   | ==         | Rakudo v2020.07+
```
Comment (JJ) [inline doc/Language/unicode_ascii.pod6:115]: Wouldn't that be Rakudo *and* Raku?

**Review by JJ — APPROVED** [review]
It's basically OK, you might want to consider using Raku instead of Rakudo.

**Comment by frithnanth** [comment]
Changed Rakudo into Raku.

**Comment by coke** [comment]
I think it should be Rakudo / it's showing which version of the compiler the feature was added in

**Comment by JJ** [comment]
> I think it should be Rakudo / it's showing which version of the compiler the feature was added in

The problem is that it's too fine a distinction for the documentation, I'm afraid. As a matter of fact, we often simply eliminate the name altogether.

**Comment by CIAvash** [comment]
`≡` is equivalent to `(==)` set operator, not `==`. Same for `≢`, which is equivalent to `!(==)`.

**Comment by JJ** [comment]
Many thanks. Reverting...

**Pull request merged**